RecoLocalCalo/HGCalRecProducers/interface/HGCalCLUEAlgo.h

```diff
@@ -19,16 +19,19 @@
 
 #include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
 
+#include "tbb/task_arena.h"
```

Comment:
Why are these includes added here when they weren't needed before?

Reply:
This is a relic of my previous template implementation. It can be safely moved to the .cc where it was.

Reply:
Which already has it, in fact. Just drop it.